lib/modules/manager/cargo/artifacts.ts

```diff
@@ -65,14 +90,49 @@ export async function updateArtifacts({
     logger.debug('No Cargo.lock found');
     return null;
   }
+
+  const isLockFileMaintenance = config.updateType === 'lockFileMaintenance';
+  if (
+    !isLockFileMaintenance &&
+    (updatedDeps === undefined || updatedDeps.length < 1)
```

Comment:
```suggestion
    (!updatedDeps?.length)
```
isn't that the same but shorter?

Reply:
I guess. It's not really obvious that this is checking for `length != 0` unless you're intimate with javascript.